pkg/handler/templates.go

```diff
@@ -293,6 +294,31 @@ func (th *TemplateHandler) deleteTemplate(c echo.Context) error {
 	return c.NoContent(http.StatusNoContent)
 }
 
+// GetTemplateRepoConfigurationFiles godoc
+// @Summary      Get configuration files for all repositories in a template
+// @ID           getTemplateRepoConfigurationFiles
+// @Tags         repositories
```

Comment:
I think this should be templates ?

Reply:
yea, i was thinking a tag of repositories kinda made sense here since the pattern we're following is the tag is what's returned in the response. i'll update this though, since what we're returning is specific to templates